rel-eng/packages/nodejs-brace

```diff
@@ -1 +1 @@
-0.10.0-1.fm1_15 nodejs-brace/
+0.10.0-2.fm1_16 nodejs-brace/
```

Comment:
How do I see that the PR includes a tag? I didn't even think it was possible to push a tag to a PR? I appreciate trying to get ahead, but I honestly think its easier to just revert the tito tag and let us do that when we do a release post merge.